lib/pbench/server/api/resources/users_api.py

```diff
@@ -92,13 +92,13 @@ def post(self):
 
         first_name = user_data.get("first_name")
         if not first_name:
-            self.logger.warning("Missing firstName field")
-            abort(400, message="Missing firstName field")
+            self.logger.warning("Missing first_name field")
+            abort(400, message="Missing first_name field")
 
         last_name = user_data.get("last_name")
         if not last_name:
-            self.logger.warning("Missing lastName field")
-            abort(400, message="Missing lastName field")
+            self.logger.warning("Missing last_name field")
+            abort(400, message="Missing last_name field")
```

Comment:
This caught me when I was adjusting my Postman queries; figured it was worth changing.